src/frontend/Player/createPlyrPlayer.ts

```diff
@@ -59,6 +59,16 @@ export const createPlyrPlayer = (
       }
     }
     xapiStatement.initialized(contextExtensions);
+
+    const playLarge: Nullable<HTMLButtonElement> = document.querySelector(
+      '.plyr .plyr__control--overlaid',
+    );
+    if (!!playLarge) {
+      // the tabIndex of this play button is set to -1. It must not be
+      // focusable when a user navigate with its keyboard. The play button in
+      // control bar is enough.
+      playLarge.tabIndex = -1;
```

Comment:
It's not enough to set `tabIndex = -1`, this will make it so it's not selectable by keyboard but it would still be read by screens readers. You also need `aria-hidden`

Reply:
:+1: